osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/UpdateOrDeleteK8sSecurityGroupMetaTaskTestData.java

```diff
@@ -0,0 +1,159 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.tasks.conformance.openstack.securitygroup;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import javax.persistence.EntityManager;
+
+import org.osc.core.broker.job.TaskGraph;
+import org.osc.core.broker.model.entities.appliance.Appliance;
+import org.osc.core.broker.model.entities.appliance.ApplianceSoftwareVersion;
+import org.osc.core.broker.model.entities.appliance.DistributedAppliance;
+import org.osc.core.broker.model.entities.appliance.VirtualSystem;
+import org.osc.core.broker.model.entities.management.ApplianceManagerConnector;
+import org.osc.core.broker.model.entities.management.Domain;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroup;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.model.entities.virtualization.VirtualizationConnector;
+import org.osc.core.broker.model.entities.virtualization.k8s.Label;
+import org.osc.core.common.virtualization.VirtualizationType;
+
+public class UpdateOrDeleteK8sSecurityGroupMetaTaskTestData {
+
+    private static final String NO_LABEL = "NO_LABEL";
+    private static final String SINGLE_LABEL = "SINGLE_LABEL";
+    private static final String MULTI_LABEL = "MULTI_LABEL";
+
+    public final static String MGR_TYPE = "SMC";
+
+    private static final int MANY = 4;
+
+    public static SecurityGroup NO_LABEL_SG = createSecurityGroup(NO_LABEL, 0);
+    public static SecurityGroup SINGLE_LABEL_SG = createSecurityGroup(SINGLE_LABEL, 1);
```

Comment:
Can you create a new test data: SINGLE_LABEL_SG_MARKED_FOR_DELETION, this will generate a graph where all the  tasks will receive a `true`